jOOQ/src/main/java/org/jooq/impl/UserImpl.java

```diff
@@ -8,14 +9,14 @@
  * @author Timur Shaidullin
  */
 public class UserImpl implements User {
```

Comment:
All implementation classes need to be final and package-private (I'll fix this)

Reply:
Thank you. I'll take this on mind in the future